releasenotes/notes/terra-nullius-7ef598626d8118c1.yaml

```
@@ -0,0 +1,29 @@
+---
+critical:
+  - |
+    You cannot upgrade in place to Qiskit 1.0.  You must begin a new virtual environment.
+
+    From Qiskit 1.0, Qiskit is comprised of exactly one Python package: ``qiskit``.  Previously,
+    as a legacy of the "component elements" of early Qiskit, the ``qiskit`` package was a
+    dependency-only "metapackage", and the core code of Qiskit was in a package called ``qiskit-terra``.
+    As Qiskit grew, the other elements split off into their own packages (such as ``qiskit-aer``)
+    until only the core was left in the metapackage.  For Qiskit 1.0, we are removing the metapackage
+    entirely, and replacing it with the actual Qiskit code.
+
+    This means that you cannot upgrade an existing installation to Qiskit 1.0.  Instead, you must
+    create a new Python virtual environment.  Using the built-in ``venv`` module, you can do (Linux
+    and Mac):
+
+    .. code-block:: bash
+
+      # Create the new environment (only once).
+      python -m venv ~/qiskit-1.0-venv
+      # Activate the environment (every session).
+      source ~/qiskit-1.0-venv/bin/activate
+      # Install Qiskit (only once).
+      pip install 'qiskit>=1.0'
+
+    On Windows, replace ``source <venv>/bin/activate`` with ``source <venv>/Scripts/activate``.
```

```suggestion
    On Windows, replace ``source <venv>/bin/activate`` with ``source <venv>\Scripts\activate``.
```

Although I guess it really depends on which windows shell you're using it'd be `<venv>\Scripts\activate.bat` if you're using cmd.exe or `PS <venv>\Scripts\Activate.ps1` if you're using powershell. This would work if you're using bash on windows I guess though.

---

This is true, I forgot people used things other than bash-likes.  Shall we do something more here?  I'm not sure what to write without going too overboard - I don't really know how most Windows users interact with `venv`.

---

When I'm debugging something on windows personally I either do it via git bash (so what you had would work) or via cmd.exe and call the `activate.bat` file. It often depends on what I'm debugging as either shell might be needed to reproduce an issue if it involves output or default character encoding.

---

But it might be better to refer to the stdlib python docs here https://docs.python.org/3/library/venv.html#how-venvs-work

---

That's a good idea - done in 6f5b066f2a.